charts/piraeus/values.yaml

```diff
@@ -100,16 +100,16 @@ operator:
     customAnnotations: {}
   satelliteSet:
     enabled: true
-    satelliteImage: quay.io/piraeusdatastore/piraeus-server:v1.18.2
+    satelliteImage: quay.io/piraeusdatastore/piraeus-server:v1.19.0
     storagePools: {}
     sslSecret: ""
     automaticStorageType: None
     affinity: {}
     tolerations: []
     resources: {}
-    monitoringImage: quay.io/piraeusdatastore/drbd-reactor:v0.7.0
+    monitoringImage: quay.io/piraeusdatastore/drbd-reactor:v0.8.0
     monitoringBindAddress: ""
-    kernelModuleInjectionImage: quay.io/piraeusdatastore/drbd9-bionic:v9.1.7
+    kernelModuleInjectionImage: quay.io/piraeusdatastore/drbd9-bionic:v9.1.8
```

Comment:
This comment is for the haController section - Does v1.9.0 continue to use old ha-controller or will it be changed to use new release of ha-controller?

Reply:
Done now, but still waiting on https://github.com/piraeusdatastore/helm-charts/pull/13 to be merged